scraper.py

```diff
@@ -0,0 +1,46 @@
+__author__ = 'shrenilpatel'
+
+import json
+import urllib2
+
+#FOOD SERVICE
+getUrl = "https://api.uwaterloo.ca/v2/foodservices/locations.json?key=a19ad7d4ee67feebd6b7caa461251302"
+#postUrl = 'https://geotaganything.herokuapp.com/tags' #ACTUAL
+postUrl = 'http://localhost:5000/tags' #LOCAL
```

Comment:
How does posting to this URL work exactly? I've never used Heroku. Do you have to somehow ssh into the server to run this Python script?